### DISCUSSION TITLE
Search field taking space as input in first letter which creates confusion

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13256] [Feature Requests]
### Describe the Bug

if we type space in the search bar as the first letter, it filters items with space i guess

if the user wrongly types space in the first letter and comes out. we will never notice it since it is not visible. user will think data is missing since search is empty

![image](https://user-images.githubusercontent.com/36895131/168093226-1f92ea23-03de-4584-8e92-c867206b5a0f.png)

### To Reproduce

just type space in search filter and come out

### Errors Shown

filters data. user will think data is missing since search is empty

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres

### What browser are you using?

Chrome

### How are you deploying Directus?

Directus cloud

**Comment by benhaynes** [comment]
Haha, good catch... that's an interesting one. Seems like a `trim` on the search query would resolve this.

**Comment by rijkvanzanten** [comment]
Moving this to a discussion as it's technically working as intended. @benhaynes any potential uses of needing to find an item with a single space? I guess you could always use the regular filters for that specific use case

**Reply by benhaynes** [reply]
I guess there could be use cases (this includes tabs, etc)... but we need to ensure the UI properly shows users that there is a search string present (can't see whitespace). Also, with this more "fuzzy" search we could always force the exact search strings to be wrapped in quotes (like Google) otherwise trim.

**Reply by rijkvanzanten** [reply]
Difference in white-space is generally visible in the existence of a placeholder, but yeah 👍🏻 

<img width="507" alt="CleanShot 2022-05-12 at 10 32 24@2x" src="https://user-images.githubusercontent.com/9141017/168099663-39c0f7cc-78e4-4dad-953a-91e70af2b991.png">

<img width="510" alt="CleanShot 2022-05-12 at 10 32 28@2x" src="https://user-images.githubusercontent.com/9141017/168099668-190608be-63a3-4f51-9be8-b68d270c09a8.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
